Billing.API/DopplerSecurity/CryptoHelper.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Billing.API.DopplerSecurity
+{
+    public class CryptoHelper
+    {
+        public static string GenerateInvoiceSign(string clientPefix, int clientId, int fileId, string hashKey)
+        {
+            var key = $"{clientPefix}{clientId}{fileId}{hashKey}";
```

Comment:
What do you think about this?

```suggestion
    public class CryptoHelper
    {
        private readonly string _hashKey;
        
        public CryptoHelper(IOptions<InvoiceProviderOptions> options)
        {
            _hashKey = options.Value.HashKey;
        }
        
        public string GenerateInvoiceSign(string clientPefix, int clientId, int fileId, string hashKey)
        {
            var key = $"{clientPefix}{clientId}{fileId}{_hashKey}";
```

Reply:
You can even make it generic:

What do you think about this?

```suggestion
    public class CryptoHelper
    {
        private readonly string _hashKey;
        
        public CryptoHelper(IOptions<CryptoHelperOptions> options)
        {
            _hashKey = options.Value.HashKey;
        }
        
        public string GenerateSignature(string payload)
        {
            var key = $"{payload}{_hashKey}";
```